lib/ansible/module_utils/network/ftd/configuration.py

```diff
@@ -28,86 +30,358 @@
 INVALID_UUID_ERROR_MESSAGE = "Validation failed due to an invalid UUID"
 DUPLICATE_NAME_ERROR_MESSAGE = "Validation failed due to a duplicate name"
 
+MULTIPLE_DUPLICATES_FOUND_ERROR = (
+    "Cannot add a new object. An object(s) with the same attributes exists."
+    "Multiple objects returned according to filters being specified. "
+    "Please specify more specific filters which can find exact object that caused duplication error")
+
+
+class OperationNamePrefix:
```

Comment:
when you use classes without rooting them at object, you should make sure that the file contains our boilerplate:

```
# Make coding more python3-ish
from __future__ import (absolute_import, division, print_function)
__metaclass__ = type
```

In particular, the ```__metaclass__ = type``` makes sure that all classes are implicitly rooted at object.  On Python2, when a class hierarchy is rooted at object, it is a new-style class with additional features.  When it is not rooted at objcet, it is an old-style class.